owncloudDomain/src/main/java/com/owncloud/android/domain/camerauploads/model/FolderBackUpConfiguration.kt

```diff
@@ -28,6 +28,8 @@ data class FolderBackUpConfiguration(
     val chargingOnly: Boolean,
     val lastSyncTimestamp: Long,
     val name: String,
+    val spaceId: String?,
+    val spaceName: String?,
```

Comment:
This must not contain the space name, it can lead to inconsistencies

Reply:
Done